arch/risc-v/src/esp32c3/esp32c3_rtc.c

```diff
@@ -2464,8 +2467,8 @@ int up_rtc_gettime(struct timespec *tp)
     }
   else
     {
-      time_us = = esp32c3_rtc_get_time_us() +
-                    esp32c3_rtc_get_boot_time();
+      time_us = esp32c3_rtc_get_time_us() +
+                esp32c3_rtc_get_boot_time();
```

Comment:
```suggestion
      time_us = esp32c3_rtc_get_time_us() + esp32c3_rtc_get_boot_time();
```
nit: These lines fit in a single one, how about merging them?

Reply:
Thanks fixed.